packages/client/src/runtime/core/types/Utils.ts

```diff
@@ -118,6 +118,20 @@ export type PayloadToResult<P, O extends Record<any, any> = RenameAndNestPayload
 
 export type Select<T, U> = T extends U ? T : never
 
+// prettier-ignore
+export type Equals<A, B> = (<T>() => T extends A ? 1 : 2) extends (<T>() => T extends B ? 1 : 2) ? 1 : 0
```

Comment:
Does not seem to be used?

Reply:
Yes, it was left over from related work that will be merged soon.